ivy/array/layers.py

```diff
@@ -445,12 +445,12 @@ def depthwise_conv2d(
     def conv2d(
         self: ivy.Array,
         filters: Union[ivy.Array, ivy.NativeArray],
-        strides: Union[int, Tuple[int], Tuple[int, int]],
+        strides: Union[int, Tuple[int, int]],
         padding: str,
         /,
         *,
         data_format: str = "NHWC",
-        dilations: Optional[Union[int, Tuple[int], Tuple[int, int]]] = 1,
```

Comment:
Why the optional type hint got removed?

Reply:
Hi @CatB1t,
1. Actually it was done long ago by someone else #975, I just changed some small details, so I thought creating an issue is kinda too much.
2. I discussed this in the `function_arguments` channel in discord, and it came down like this:
> Optional should only be used in the typehints if the default value of the argument is None